deployment/v3/utils/prop_migrator/README.md

```diff
@@ -0,0 +1,225 @@
+# Repository Property File Comparator
+
+This script compares property files between an old repository and a latest repository, identifies the differences, and generates CSV files to summarize the results.
+
+## Overview
+
+When working with version-controlled repositories, it's common to have different versions of property files that contain configuration settings. This script helps you compare property files between two repositories (old and latest) and provides insights into the differences between them. It is particularly useful when migrating from an old version of a repository to a new version, ensuring that important configuration settings are maintained and updated correctly.
+
+The script performs the following steps:
+
+1. Cloning repositories: It clones the old and latest repositories using Git, allowing you to specify the repository URLs and branch names.
```

Comment:
if the folder is already present then it runs against the folder and generates csv files and updates the property files only if the property files are not already updated, if it is updated then it will do nothing.